core/src/upickle/core/internal/LinkedHashMap.scala

```diff
@@ -0,0 +1,54 @@
+package upickle.core.internal
+
+import upickle.core.compat.Factory
+
+import scala.collection.mutable
+import java.{util => ju}
+
+class LinkedHashMap[K, V] private (underlying: ju.LinkedHashMap[K, V])
+    extends mutable.Map[K, V]
+    with mutable.Builder[(K, V), LinkedHashMap[K, V]]
+    with LinkedHashMapCompat[K, V]
+    {
```

Comment:
Is this a copy of somewhere else or does it need a thorough review?

Some ScalaDoc would be nice. It should at least tell that it is a mutable collection.

Reply:
I wrote it from scratch. So needs a proper review.
The implementation is internal to this repository, also it extends `scala.collection.mutable.Map`.
Nevertheless I can add the Scaladoc.